source/brailleTables.py

```diff
@@ -69,6 +77,32 @@ class BrailleTable(NamedTuple):
 and cleared when calling L{terminate}.
 """
 
+_inputTableForLangs = dict()
+"""Maps languages to input L{BrailleTable.fileName}."""
+
+_translationTableForLangs = dict()
+"""Maps languages to translation L{BrailleTable.fileName}."""
+
+
+def getDefaultTableForCurLang(tableType: TableType) -> str:
+	"""Gets the file name of the braille table for the current NVDA language.
+	:param tableType: input or translation.
+	:return: A L{BrailleTable} fileName.
+	"""
+	if tableType == TableType.INPUT:
```

Comment:
Could you make this a match statement instead? I think it's nicer syntax, though I don't insist.